input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
```

Comment:
Commenting here to get an inline comment. 

Please update profile title and link in the Terminology page.

Reply:
Amended, thanks.